webrtc.html

```diff
@@ -3739,6 +3769,25 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </section>
 
       <section>
+        <h3>RTCRtpTransceiver Interface</h3>
+
+        <dl class="idl" title="dictionary RTCRtpTransceiverInit">
+          <dt>sequence&lt;RTCRtpEncodingParameters&gt; sendEncodings</dt>
```

Comment:
Why is this necessary here?  Can't you just set parameters (`setParameters()`) on the sender?